### PR TITLE
Update analysis_options

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

File: app_flutter/lib/state/agent.dart [hunk 1]
```diff
@@ -81,7 +81,7 @@ class AgentState extends ChangeNotifier {
           .fetchAgentStatuses()
           .then((CocoonResponse<List<Agent>> response) {
         if (response.error != null) {
-          print(response.error);
+          print(response.error); // TODO(ianh): report this rather than printing it
```
Comment (CaseyHillers) [inline app_flutter/lib/state/agent.dart]: I believe these TODOs should also link to https://github.com/flutter/flutter/issues/52697 (I don't know if flutter/cocoon has that enabled for the linter)

Comment (Hixie) [reply app_flutter/lib/state/agent.dart]: Oh interesting I was imagining they'd just get logged to a snackbar or some such. Logging seems good too though.

Comment (Hixie) [reply app_flutter/lib/state/agent.dart]: updated the TODO, will commit on green

**Comment by Hixie** [comment]
Analysis failed because dartfmt tried to reformat the lines I made more readable; I have a PR to switch us to 120 character lines (https://github.com/flutter/cocoon/pull/699) which should be a reasonable compromise between the readability we can get without a formatter and the consistency we can get with a formatter.

**Pull request merged**